test/AotCompile/basic_tcp_ops.mlir

```diff
@@ -23,3 +23,8 @@ func.func @func_3(%arg0: tensor<f32>,
   %0 = tcp.add %arg0_bcast, %arg1 : tensor<?xf32>, tensor<?xf32> -> tensor<?xf32>
   return %0 : tensor<?xf32>
 }
+
+func.func @func_round(%arg0: tensor<?x?xf32>) -> tensor<?x?xf32> {
```

Comment:
For consistency I suggest calling this `func_4` or renaming the previous ones to have a descriptive name like `func_mul_add_1` etc.

Reply:
Sure, done.